### ISSUE TITLE
Release of model weights

### ISSUE THREAD

**Issue by LudvigDillen** [circle-group/cumperlay#2]
Hi,

Would it be possible for you to release the model weights of your models, specifically the ones trained on ISIC2018? I tried running the following
Command: OMP_NUM_THREADS=1 python -u train.py --config config/isic-swin-silv2-inpd-multreg-enp.yaml --test --gpu 0
and got the results:

      test/accuracy         0.4166666567325592
        test/loss            2.399796962738037
      test/loss_ce           1.875800609588623
     test/loss_ce_w          1.875800609588623
  test/loss_cfn_entropy      7.643303871154785
 test/loss_cfn_entropy_w  -0.0007643306162208319
 test/loss_multifilt_reg    2.9619715213775635
test/loss_multifilt_reg_w  0.029619721695780754
    test/loss_topo_ce       1.9805625677108765
   test/loss_topo_ce_w      0.4951406419277191
      test/roc_auc          0.45845258235931396
   test/topo_accuracy       0.05886243283748627
    test/topo_roc_auc        0.52615886926651

I guess this is just from the pre-trained Swin2 backbone, and that I'd have to re-train your model to get up to the performance in the paper.

Thanks for the great work!
Ludde

**Comment by Kausta** [comment]
Hello,

Thank you for your interest in the work. I uploaded the model weights trained on ISIC2018 to [experiments/isic2018/best_val.pt](https://github.com/circle-group/cumperlay/blob/298e8ff3ffded5286d6cd3e6f75920be1e7d2c00/experiments/isic2018/best_val.pt), along with the corresponding config file and the command (which should match [config/isic-swin-silv2-inpd-multreg-enp.yaml](https://github.com/circle-group/cumperlay/blob/298e8ff3ffded5286d6cd3e6f75920be1e7d2c00/config/isic-swin-silv2-inpd-multreg-enp.yaml)). The results on this file should match:
```
	test/accuracy         0.8346560597419739
        test/loss           1.0702425241470337
	test/roc_auc           0.965003490447998
   	test/topo_accuracy       0.8320105671882629
    	test/topo_roc_auc        0.944585382938385
```
I will upload the remaining weights as well, but it may take me a bit of time. If you need any help or run into any issues, please let me know.
Thanks,

Caner

**Comment by LudvigDillen** [comment]
Hi there,

Thanks for the weights of TopoSwin-MP. I managed to get these results which largely overlaps with the paper :)
```
   test/accuracy 0.8320105671882629 (paper has 0.8347, ok)
   test/loss 1.0674318075180054
   test/roc_auc 0.9648759365081787 (paper has 0.9650, ok)
   test/topo_accuracy 0.8339946866035461
   test/topo_roc_auc 0.9449227452278137
```
Although, I did not understand the command (cmd.txt) you added since it was with the train flag on. Also, I had to debug a bit to realize that I had to add this line to the config file
`resume: experiments/isic2018/best_val.pt
`
But after doing that, everything worked out. Would be very appreciated if you uploaded the weights for "TopoSwin-Betti" and "TopoSwin-SP" for ISIC2018 as well. 

I guess "TopoSwin-SP" belongs to the config `isic-swin-silv2-inpd-singreg-enp.yaml` but which config belong to "TopoSwin-Betti"? And what is the difference between "inpd" and "inponly"?

All the best,
Ludde
